app/routes.py

```diff
@@ -22,13 +23,16 @@ def index():
 #Create challenge page
 @app.route('/create', methods=['GET', 'POST'])
 def create():
+    updateDB()
```

Comment:
Do we need to updateDB in create? and most places that arent directly related to an existing challenge (creating account, logging in, logging out?)

Reply:
Ah, probably not, I'll remove it there.